docs/settings.md

```diff
@@ -0,0 +1,56 @@
+# Settings
+
+Copier settings are stored in `<CONFIG_ROOT>/settings.yml` where `<CONFIG_ROOT>` is the
```

Comment:
Should be rename `<CONFIG_ROOT>` to `<COPIER_HOME>` for better clarity and consistency with the code block title `$COPIER_HOME/settings.yml` below?

Reply:
I vote for `<CONFIG_ROOT>` everywhere, or `<CONFIG_FOLDER>`. `$COPIER_HOME` makes it sound like that's a valid environment variable.

Reply:
Fixed, `<CONFIG_ROOT>` everywhere 👍🏼